e2e/cypress/integration/menus/status_dropdown_spec.js

```diff
@@ -181,4 +182,33 @@ describe('Status dropdown menu', () => {
                 and('have.text', 'Status').and('not.have.css', 'cursor', 'pointer');
         });
     });
+
+    describe('Custom status option in Status dropdown menu', () => {
+        before(() => {
+            cy.apiUpdateConfig({TeamSettings: {EnableCustomUserStatuses: true}});
+            cy.apiClearUserCustomStatus();
+        });
+
+        it('should show Set a Custom Status header when EnableCustomStatuses option is enabled, with cursor pointer', () => {
+            // # Wait for posts to load
+            cy.get('#postListContent').should('be.visible');
+
+            // # Open status menu
+            cy.get('.MenuWrapper .status-wrapper.status-selector button.status').click();
+
+            // # Verify "Set a Custom Status" header is visible
+            cy.get('.MenuWrapper.status-dropdown-menu .Menu__content.dropdown-menu li#status-menu-custom-status').should('be.visible').
+                and('have.text', 'Set a Custom Status').and('have.css', 'cursor', 'pointer');
+        });
+
+        specify('clear button should not be visible if status is not set', () => {
```

Comment:
What is `specify` used for? Is it the same as `it`?

Reply:
Yes. Both `specify` and `it` have the same meaning.

The reason `specify` was used here instead of `it` was because starting the sentence with `clear button should...` with `it` was not meaningful (i.e. `it clear button should...`).

Reply:
Oh I see. I didn't realize that was the reason some testing frameworks used `it` to begin with. That's actually kind of nice if we were consistent with it. Thanks for changing it to `it` though to keep that.